bug_report.php

```diff
@@ -322,16 +322,11 @@
 		}
 	}
 }
-?>
-
-<div class="col-md-12 col-xs-12">
-	<div class="space-10"></div>
-	<div class="alert alert-success">
 
-<?php
-echo '<p>' . lang_get( 'operation_successful' ) . '</p><br />';
-print_button( string_get_bug_view_url( $t_bug_id ), sprintf( lang_get( 'view_submitted_bug_link' ), $t_bug_id ) );
-print_button( 'view_all_bug_page.php', lang_get( 'view_bugs_link' ) );
+html_operation_confirmation( array(
+	array( string_get_bug_view_url( $t_bug_id ), sprintf( lang_get( 'view_submitted_bug_link' ), $t_bug_id ) ),
+	array( 'view_all_bug_page.php', lang_get( 'view_bugs_link' ) ),
+) );
```

Comment:
> Should this be a call to html_operation_successful instead?

html_operation_successful() only allows a single button, so to maintain existing functionality it is required to use html_operation_confirmation().

> I find this function call with array of arrays input to be rather complex. 
> I would suggest separating the buttons build up logic first. 

So you're saying I should do this instead ?
```
$t_buttons = array(
	array( string_get_bug_view_url( $t_bug_id ), sprintf( lang_get( 'view_submitted_bug_link' ), $t_bug_id ) ),
	array( 'view_all_bug_page.php', lang_get( 'view_bugs_link' ) ),
);
html_operation_confirmation( $t_buttons );
```
If that's the case, I don't really see the benefit.

> Also using static arrays if possible http://php.net/SplFixedArray

Considering that in 100% of current use cases the array size is <= 2, and that it's not reused after the function call, I'm not sure there is a real benefit for fixed-size arrays here.